src/blockdata/transaction.rs

```diff
@@ -223,6 +224,18 @@ pub struct TxIn {
     pub witness: Witness
 }
 
+impl TxIn {
+    /// Returns true if a transactions nLockTime is enabled by this input.
+    ///
+    /// If this returns false and OP_CHECKLOCKTIMEVERIFY is used in the redeem script with this
+    /// input then the script execution will fail [BIP-0065].
+    ///
+    /// [BIP-65](https://github.com/bitcoin/bips/blob/master/bip-0065.mediawiki)
+    pub fn enables_n_lock_time(&self) -> bool {
```

Comment:
I don't think we should use `n` in these names here. `nLockTime` is the name of a variable in Core which even uses Hungarian notation - unidiomatic in Rust. The thing it refers to is in normal speech called just "lock time". So it makes sense to remove `n_` in Rust.

Reply:
Seems like historically in Bitcoin technical community there was not distinction between Bitcoin Core code details and name of concepts. This is going to hard to break habit.

Reply:
This function got renamed but there is still a bunch of `n_lock_time` identifiers (variables and parameters). The justification for these I would propose is that a `LockTime` can be either the transaction lock time field (called `lock_time` in Rust and `nLockTime` in Bitcoin Core) or the script argument to `n OP_CHECKLOCKTIMEVERIFY`. I was hoping that using `n_lock_time` would help to make it explicit that the variable is meant to be the transaction field.